airflow/www/static/connection_form.js

```diff
@@ -7,7 +7,12 @@
         jdbc: {
             hidden_fields: ['port', 'schema', 'extra'],
             relabeling: {'host': 'Connection URL'},
+        },
+        google_cloud_platform: {
```

Comment:
(note, this is a forward looking comment, for when we migrate BQ to the new gcloud based hook)